core/src/main/scala/akka/persistence/cassandra/query/EventsByTagStage.scala

```diff
@@ -421,7 +421,7 @@ import scala.compat.java8.FutureConverters._
         }
 
         def optionallySchedule(key: Any, duration: Option[FiniteDuration]): Unit = {
-          duration.foreach(fd => schedulePeriodically(key, fd))
+          duration.foreach(fd => scheduleWithFixedDelay(key, FiniteDuration(0L, SECONDS), fd))
```

Comment:
to keep same initial interval as before this should be `fd, fd`

Reply:
Fixed